schemas/com.mailgun/message_bounced/jsonschema/1-0-0

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message bounced event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_bounced",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "attachment-count": {
+      "description": "Number of attachments in attachment-x (where x is a number) fields.",
+      "type": "string",
+      "maxLength": 8
+    },
+    "event": {
+      "description": "Event name ('bounced').",
+      "type": "string",
+      "maxLength": 16
+    },
+    "recipient": {
+      "description": "Recipient who could not be reached.",
+      "type": "string",
+      "maxLength": 512
+    },
+    "domain": {
+      "description": "Domain that sent the original message.",
+      "type": "string",
+      "maxLength": 256
+    },
+    "message-headers": {
+      "description": "String list of all MIME headers of the original message dumped to a JSON string (order of headers preserved).",
+      "type": "string"
+    },
+    "code": {
+      "description": "SMTP bounce error code in form (X.X.X).",
+      "type": "string",
+      "maxLength": 8
+    },
+    "error": {
+      "description": "SMTP bounce error string.",
+      "type": "string"
+    },
+    "notification": {
+      "description": "Detailed reason for bouncing (optional).",
+      "type": "string"
+    },
+    "campaign-id": {
+      "description": "The id of campaign triggering the event.",
+      "type": "string"
+    },
+    "campaign-name": {
+      "description": "The name of campaign triggering the event.",
+      "type": "string"
+    },
+    "tag": {
+      "description": "Message tag, if it was tagged. See Tagging.",
+      "type": "string"
+    },
+    "mailing-list": {
+      "description": "The address of mailing list the original message was sent to.",
+      "type": "string"
+    },
+    "Message-Id": {
+      "description": "The Id of the message that bounced.",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "timestamp": {
+      "description": "Number of seconds passed since January 1, 1970 (see securing webhooks).",
+      "type": "string",
+      "maxLength": 16
```

Comment:
I believe each of the Mailgun schemas has this `timestamp` field. In the Scala adapter, we should be proactively changing it into a ISO8601 datetime (and updating the schema to match). Then it will appear in Redshift as a first-class `TIMESTAMP` column.